lib/pymedphys/tests/mosaiq/create_mock_data.py

```diff
@@ -11,6 +14,20 @@
 sa_user = "sa"
 sa_password = "sqlServerPassw0rd"
 
+# vary the number of fractions a bit
+number_of_fractions = [20, 25, 30]
```

Comment:
Just a nit-pick, more just preference.

```suggestion
NUMBER_OF_FRACTIONS = (20, 25, 30)
```

Making the top level constant use all-caps to symbolise constant, and utilise `tuple` for constants so it is clear that the variable won't be mutated.